src/unstract/adapters/llm/mistral/src/static/json_schema.json

```diff
@@ -25,8 +14,10 @@
       "description": "Provide a unique name for this adapter instance. Example: mistral-group-1"
     },
     "model": {
+      "type": "string",
       "title": "Model",
-      "$ref": "#/definitions/models"
+      "default": "mistral-large-latest",
+      "description": "Provide the model name to be used. Example: mistral-tiny, mistral-small, mistral-medium, mistral-large-latest"
```

Comment:
NIT: The examples would help, alternatively we can also link to mistral docs on supported models - but we need to check if its doable with JSON schema and our current implementation